### ISSUE TITLE
Seeing $null in current play store build

### ISSUE THREAD

**Issue by binamov** [dougkeen/BartRunnerAndroid#12]
The release note said it should be fixed. Anything else I need to do?

**Comment by binamov** [comment]
2.2.8 running on Android 6.0.1

**Comment by dougkeen** [comment]
Looks like BART is having problems on their end again (I've notified them and hopefully they'll have it resolved soon). I'll try and get a better error message in the app, though.

**Comment by binamov** [comment]
Aha... Interesting that other apps work though... Maybe they're looking at a different endpoint at Bart...

**Comment by dougkeen** [comment]
Yeah, they might be using the [GTFS-RT feed](http://www.bart.gov/schedules/developers/gtfs-realtime) instead of the ["Real BART API"](http://www.bart.gov/schedules/developers/api). The latter is definitely acting screwy right now. All their ETD endpoints are currently returning something similar to this:

```
<root>
<uri>
<![CDATA[ http://api.bart.gov/api/etd.aspx?cmd=etd&orig=RICH ]]>
</uri>
<date>11/02/2016</date>
<time>03:48:17 PM PDT</time>
<station>
<name>Richmond</name>
<abbr>RICH</abbr>
<message>
<error>Updates are temporarily unavailable.</error>
</message>
</station>
<message/>
</root>
```
